lib/android/app/src/main/java/com/reactnativenavigation/utils/Context.kt

```diff
@@ -9,7 +9,8 @@ import com.reactnativenavigation.NavigationApplication
 fun Context.isDebug(): Boolean {
     return (applicationContext as ReactApplication).reactNativeHost.useDeveloperSupport
 }
-fun isDarkMode() = NavigationApplication.instance.isDarkMode()
+fun isDarkMode() = false
+//fun isDarkMode() = NavigationApplication.instance.isDarkMode()
```

Comment:
need to figure out how to stop calling this or override it. this is library code

Reply:
If necessary, we could patch the package or create our own fork for this change.